config/routes.rb

```diff
@@ -29,27 +29,27 @@
   delete "route/:id", to: 'route#destroy', as: :route_delete
   get 'route/showEditRoute/:id', to: 'route#showEditRoute', as: :route_show_edit_route
   get "route/routeDetails/:id", to: 'route#routeDetails', as: :route_route_details
-  
-  post 'route/removeRouteBusstop/:id', to: 'route#removeRouteBusstop', as: :route_remove_route_busstop 
-  post 'route/includeRouteBusstop/:id', to: 'route#includeRouteBusstop', as: :route_include_route_busstop 
-  
+
+  post 'route/removeRouteBusstop', to: 'route#removeRouteBusstop', as: :route_remove_route_busstop
+  post 'route/includeRouteBusstop', to: 'route#includeRouteBusstop', as: :route_include_route_busstop
+
   post 'route/saveBusStopOnRoute', to: 'route#saveBusStopOnRoute'
   get 'welcome', to: 'welcome#welcome', as: :welcome_welcome
 
-  root 'front_ends#index'
+  root 'welcome#welcome'
   #rogue lines
-  
+
   #
-  
+
   devise_scope :admin do
 
     #get 'admin/passwords/new', to: 'devise/admin/passwords#new', as: :admin_passwords_new
     get 'admin/index', to: 'devise/admin/registrations#index', as: :admin_index
     get 'admin/pendingUser', to: 'devise/admin/registrations#pendingUser', as: :admin_pending_user
     post 'admin/approveUser/:id', to: 'devise/admin/registrations#approveUser', as: :admin_approve_user
-    post 'admin/approveAdmin/:id', to: 'devise/admin/registrations#approveAdmin', as: :admin_approve_admin 
-    get 'unApprovedAccount', to: 'devise/admin/registrations#unApprovedAdmin', as: :admin_unapproved_account 
-    get 'userProfile/:id', to: 'devise/admin/registrations#userProfile', as: :admin_user_profile 
+    post 'admin/approveAdmin/:id', to: 'devise/admin/registrations#approveAdmin', as: :admin_approve_admin
```

Comment:
Line is too long. [106/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.